src/functional/swing_slide_filter.zig

```diff
@@ -174,19 +189,223 @@ pub fn compressSwing(
 
         // Get optimal slope that minimizes the squared error Eq. (5).
         const slope: f80 = @max(
-            @min(slope_derivate / sum_square, upper_bound_linear_function.slope),
-            lower_bound_linear_function.slope,
+            @min(slope_derivate / sum_square, upper_bound.slope),
+            lower_bound.slope,
```

Comment:
Well, the line has 97 characters without counting the 8 spaces at the beginning. Is still confusing if that is allowed or not. Just in case I will break the line.

Reply:
My bad, I review the code on an IPad using the GitHub Mobile app, so I over looked that the line was part of a function call.